### DISCUSSION TITLE
Using layouts with vue script setup syntax?

### DISCUSSION THREAD

**Discussion by isaac-souza** [inertiajs/inertia#1128] [Help (Vue)]
Hi,

How can I define a persistent layout when using the script setup syntax in vue 3? Is is possible?

Couldn't find anything in the docs

```vue 
<script>
    import { defineComponent } from 'vue'
    import { useForm, usePage } from '@inertiajs/inertia-vue3'
    import AppLayout from '@/Layouts/App.vue'
    import PageHeader from '@/Components/PageHeader.vue'
    import Card from '@/Components/Card.vue'
    
    export default defineComponent({
        name: 'AccountEdit',
        layout: AppLayout,
        components: {
            PageHeader,
            Card,
        },
```

Appreciate any help, thanks.

BTW, I damn love Inertia.js, it is freaking awesome.

Isaac

**Comment by ajnsn** [comment]
https://github.com/inertiajs/inertia/discussions/651
https://github.com/inertiajs/inertia/discussions/481
https://github.com/inertiajs/inertia/discussions/873
